js/src/modals/CreateWallet/createWalletStore.js

```diff
@@ -283,7 +283,8 @@ export default class CreateWalletStore {
 
     const owners = _wallet.owners.filter((owner) => !/^(0x)?0*$/.test(owner));
 
-    if (_wallet.required > owners.length) {
+    // Real number of owners is owners + creator
+    if (_wallet.required > owners.length + 1) {
```

Comment:
we should now check that there are no duplicate identities between the creator and named owners.

Reply:
Is the actual wallet code ensuring that?

Reply:
not sure about the very latest version, but i know at least one of the original versions didn't.

Reply:
At least here it doesn't : https://github.com/paritytech/contracts/blob/master/Wallet.sol#L58
We could enforce this UI-wise though, even if it's not enforced in the underlying contract.

Reply:
Done